### ISSUE TITLE
A11y: as a student, particularly with visual or motor impairments, I want to be able to press enter to go to the next step

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#858]
concerns: 
-how does this affect the idea of pressing enter to drop down? 
maybe shift enter to drop down and enter to do new step? 
that is consistent with most text based systems where enter is a separate submit function
-how does this affect the UI? 
actually if you incorporate the shift enter it doesn't have to change the UI at all, just how you work through it using keyboard commands (with better flow for a screenreader user) 
-potentially option to dismantle the GUI 

#554: enter should do a new line
how would enter/shift enter come across to mainstream students? 

-also consider: keyboard navigation of reading out the math (up and down arrows, etc)

**Comment by jscholes** [comment]
@sinabahram or @NSoiffer, are there any accessibility implications here re: Enter or Shift+Enter already having a Math/navigation-related function within the editor?  I know chunking isn't in there yet, and even if it was, that would be Arrow-oriented, right?

**Comment by jscholes** [comment]
Related: #859

**Comment by sinabahram** [comment]
With the editor is a Neil question; however, within the exploration of mathematics, both enter and shift+enter are already defined; however, you are asking directly about being within the edit field of the MathLive component, and my understanding is that this was used for submission, unless if multiline mathematics is allowed.
